### ISSUE TITLE
PWA Support

### ISSUE THREAD

**Issue by taha-mcp** [markusfisch/PieLauncher#26]
This app is amazing and I love the idea behind it. The only thing that is stopping me from using this launcher is the lack of PWA support. I have a few PWA apps that I use regularly and I can't go without them.

**Comment by markusfisch** [comment]
Yes, I know that's definetly an issue 😬 It's already requested in https://github.com/markusfisch/PieLauncher/issues/2 

Unfortunately, it's not trivial to add support for widgets, which is what web links on the home screen really are.
